components/nexus-mime/src/main/resources/org/apache/tika/mime/custom-mimetypes.xml

```diff
@@ -15,14 +15,4 @@
 -->
 <mime-info>
 
```

Comment:
No particular reason at all. We can remove the file if we want, it's controlled and picked up by Tika config anyway (ie. removing it would not need any change on our side).
OTOH, given the frequent changes done in this file, I just thought it might be still useful (instead of codereading Tika where the file needs to be next time we need some override) :smile: